core/licenses/jna-4.4.0.jar.sha1

```diff
@@ -1 +1 @@
-6edc9b4514969d768039acf43f04210b15658cd7
\ No newline at end of file
+d85e9ceee4d97e738524a231687ff28446376c97
```

Comment:
I'm confused by this change? How is the SHA changing without the version? And I think we shouldn't change the version unless upstream has incremented the version and we are incrementing with them.